catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/validation/impl/MatchOneValidatorTest.java

```diff
@@ -0,0 +1,140 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.validation.impl;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Matchers;
+
+import com.google.common.collect.ImmutableSet;
+
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.validation.impl.report.AttributeValidationReportImpl;
+import ddf.catalog.validation.impl.validator.EnumerationValidator;
+import ddf.catalog.validation.impl.validator.MatchOneValidator;
+import ddf.catalog.validation.impl.validator.SizeValidator;
+import ddf.catalog.validation.report.AttributeValidationReport;
+
+public class MatchOneValidatorTest {
+
+    private static final AttributeImpl VALID_ATTRIBUTE_1 = new AttributeImpl(Core.TITLE, "bob");
+
+    private static final AttributeImpl VALID_ATTRIBUTE_2 = new AttributeImpl(Core.TITLE, "bobby");
+
+    private static final AttributeImpl INVALID_ATTRIBUTE = new AttributeImpl(Core.TITLE, "robert");
+
+    private static final Set<String> VALID_ENUMERATIONS = ImmutableSet.of("bob", "rob", "bobby");
+
+    private static final Set<String> VALID_ENUMERATIONS_2 = ImmutableSet.of("ben");
+
+    private MatchOneValidator matchOneValidator;
+
+    private EnumerationValidator stubEmptyValidator;
+
+    private EnumerationValidator enumerationValidator;
+
+    private EnumerationValidator enumerationValidator2;
+
+    @Before
+    public void setUp() {
+        stubEmptyValidator = mock(EnumerationValidator.class);
+        when(stubEmptyValidator.validate(Matchers.any(AttributeImpl.class))).thenReturn(Optional.of(
+                new AttributeValidationReportImpl()));
+
+        enumerationValidator = new EnumerationValidator(VALID_ENUMERATIONS, false);
+        enumerationValidator2 = new EnumerationValidator(VALID_ENUMERATIONS_2, false);
+        SizeValidator sizeValidator = new SizeValidator(1, 3);
+        matchOneValidator = new MatchOneValidator(Arrays.asList(enumerationValidator,
+                sizeValidator));
+    }
+
+    @Test
+    public void testNullValidatorList() {
```

Comment:
Good test names generally include the state being tested and the expected result/behavior. So this test method could be something like `testMatchOneValidatorPassesWhenValidatorListIsNull`.